CAIPs/caip-275.md

```diff
@@ -186,11 +187,13 @@ To further clarify different authentication flow configurations we can consider
 }
 ```
 
-As this authentication flow calls for the `injected` extension, without specifying which one, it should use EIP-6963 to check if a matching extension provider is available, and use that if it is available. If one is not found, we resolve to the default one.
-For ethereum addresses, this would be `window.ethereum`.
-However, this configuration is not recommended as having multiple wallets generates a race condition and can provide false positives where there is a wallet extension but it does not have the account linked to the domain. We recommend EIP-6963 if possible.
+As this authentication flow calls for the `injected` extension, without specifying which one, it should use the default wallet provider at the browser.
+For EVM chains, this would be `window.ethereum`.
```

Comment:
I'm still a little confused why this flow skips over EIP 6963 signaling (which could check for `com.wallet.rdns` by name if multiple extensions are installed) and defaulting to legacy EIP 1193 (which has that famous race condition to inject the provider, and could open a different wallet than intended if another wallet overwrote window.ethereum).

Reply:
it's not really a blocker here, i can just open a separate issue